dhqa_data.py

```diff
@@ -58,11 +64,8 @@ def get_post_info(div, topic_url, feed):
     [comment.extract() for comment in threadpost.findAll(
         text=lambda text:isinstance(text, Comment))]
 
-    # get post html content
-    info['html_content'] = threadpost.div.prettify()
-
-    # extract text from post on html page
-    info['content'] = threadpost.div.get_text()
+    # get post content
+    info['content'] = threadpost.div.prettify()
```

Comment:
This update means that 'content' will contain html tags, making text analysis difficult (essentially would need to use BeautifulSoup again). Is there a reason for this change?

Reply:
No, my mistake — I lost some of your changes somehow! I thought I caught them, but I obviously missed this one. Thanks for flagging.

Reply:
no worries! I saw merge away then 👍